sdlib/d/gc/tcache.d

```diff
@@ -171,6 +231,29 @@ public:
 	}
 
 private:
+	// Force large allocation rather than slab
+	size_t upsizeToLarge(size_t size) {
+		auto aSize = size;
+		while (getAllocSize(aSize) <= SizeClass.Small) {
```

Comment:
none of the functions require looping in any way. This code is going to be called on every allocations, why is this looping?